README.md

```diff
@@ -186,48 +186,53 @@ data class Responses(
-|                              |   `QUARKUS_REFLECTION` - This option adds @RegisterForReflection to the generated models. Requires dependency "'io.quarkus:quarkus-core:+" |
-|                              |   `MICRONAUT_INTROSPECTION` - This option adds @Introspected to the generated models. Requires dependency "'io.micronaut:micronaut-core:+" |
-|                              |   `MICRONAUT_REFLECTION` - This option adds @ReflectiveAccess to the generated models. Requires dependency "'io.micronaut:micronaut-core:+" |
-|                              |   `INCLUDE_COMPANION_OBJECT` - This option adds a companion object to the generated models. |
-|                              |   `SEALED_INTERFACES_FOR_ONE_OF` - This option enables the generation of interfaces for discriminated oneOf types |
-|   `--output-directory`       | Allows the generation dir to be overridden. Defaults to current dir |
-|   `--resources-path`         | Allows the path for generated resources to be overridden. Defaults to `src/main/resources` |
-|   `--src-path`               | Allows the path for generated source files to be overridden. Defaults to `src/main/kotlin` |
-|   `--targets`                | Targets are the parts of the application that you want to be generated. |
-|                              | CHOOSE ANY OF: |
-|                              |   `HTTP_MODELS` - Jackson annotated data classes to represent the schema objects defined in the input. |
-|                              |   `CONTROLLERS` - Spring / Micronaut annotated HTTP controllers for each of the endpoints defined in the input. |
-|                              |   `CLIENT` - Simple http rest client. |
-|                              |   `QUARKUS_REFLECTION_CONFIG` - This options generates the reflection-config.json file for quarkus integration projects |
-|   `--type-overrides`         | Specify non-default kotlin types for certain OAS types. For example, generate `Instant` instead of `OffsetDateTime` |
-|                              | CHOOSE ANY OF: |
-|                              |   `DATETIME_AS_INSTANT` - Use `Instant` as the datetime type. Defaults to `OffsetDateTime` |
-|                              |   `DATETIME_AS_LOCALDATETIME` - Use `LocalDateTime` as the datetime type. Defaults to `OffsetDateTime` |
-|   `--validation-library`     | Specify which validation library to use for annotations in generated model classes. Default: JAVAX_VALIDATION |
-|                              | CHOOSE ONE OF: |
-|                              |   `JAVAX_VALIDATION` - Use `javax.validation` annotations in generated model classes (default) |
-|                              |   `JAKARTA_VALIDATION` - Use `jakarta.validation` annotations in generated model classes |
+| Parameter                | Description                                                                                                                                                                |
```

Comment:
Was this updated table generated via: `./gradlew printCodeGenUsage` ?

Reply:
aah wow cool! I'll replace the table with the generated version